runatlantis.io/docs/server-configuration.md

```diff
@@ -280,6 +280,13 @@ and set `--autoplan-modules` to `false`.
   This means that an attacker could spoof calls to Atlantis and cause it to perform malicious actions.
   :::
 
+### `--checkout-depth`
+  ```bash
+  atlantis server --checkout-depth=50
+  ```
+  How many commits from branches to fetch. Used only if `--checkout-strategy=merge`.
```

Comment:
```suggestion
  How many commits from branches to fetch. Used only if `--checkout-strategy=merge` since the `branch` checkout strategy always defaults to a shallow clone using a depth of 1.
```